<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ComicReaderWindow</class>
 <widget class="QMainWindow" name="ComicReaderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>822</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ComicReader</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="comicImage">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>651</width>
      <height>731</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="previousPageButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>37</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="nextPageButton">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>37</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="OnePageButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>37</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="DoublePageButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>37</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="BookmarkButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>37</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QToolButton" name="b_prevpageButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>37</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="b_nextpageButton">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>360</y>
      <width>37</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLabel" name="totalPage">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>20</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="currentPage">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>20</y>
      <width>31</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Agency FB</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QToolButton" name="gotoButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>37</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>go</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QComboBox" name="BookmarkList">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>20</y>
      <width>37</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="comicImage_1">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>325</width>
      <height>731</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="comicImage_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>325</width>
      <height>731</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
